### ISSUE TITLE
Sitios para comer y cenar 

### ISSUE THREAD

**Issue by jhq1002** [dgs1016/Practica1-Software#11]
Búsqueda de lugares para comer y cenar relación calidad/precio.

**Comment by jhq1002** [comment]
¿Jaime has encontrado alguno que esté bien?

**Comment by jmm1053** [comment]
Sí, encontré esta información en un foro viajero: Praga, República Checa:

La Degustation Bohême Bourgeoise: Con una estrella Michelin, este restaurante ofrece una experiencia gastronómica única, con platos tradicionales checos reinterpretados de manera moderna.
Field Restaurant: Reconocido por su enfoque en ingredientes locales y temporada, este restaurante ofrece una cocina creativa y deliciosa en un ambiente elegante y acogedor.
Lokál Dlouhááá: Si buscas una auténtica experiencia checa, Lokál Dlouhááá es el lugar adecuado. Sirven comida casera tradicional en un ambiente animado y informal.
Grecia:

Selene: Situado en la isla de Santorini, Selene ofrece una cocina gourmet griega con un enfoque en los ingredientes locales y las recetas tradicionales.
Varoulko Seaside: Ubicado en Atenas, este restaurante con estrella Michelin se especializa en cocina griega contemporánea con un enfoque en los mariscos frescos y los productos locales.
To Kati Allo: Este acogedor restaurante en Tesalónica es conocido por su excelente servicio y su deliciosa comida griega casera, especialmente sus platos de pescado fresco.
Bosnia y Herzegovina:

Inat Kuća: Ubicado en Sarajevo, este restaurante ofrece auténtica cocina bosnia en un ambiente histórico con vistas al río Miljacka.
Noovi: Situado en Mostar, Noovi es conocido por su cocina innovadora que combina influencias bosnias y mediterráneas en platos creativos y deliciosos.
Restaurant Žara: Este restaurante en Banja Luka es famoso por sus asados a la parrilla y su ambiente acogedor, perfecto para disfrutar de platos tradicionales bosnios con amigos y familiares.
